Makefile

```
@@ -23,7 +23,7 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # Image URL to use all building/pushing image targets
 IMG ?= $(REGISTRY)/$(ORG)/$(OPERATOR_NAME):v$(VERSION)
 
-all: docker-build docker-push
+all: docker-build bundle bundle-build
```

We can do one more thing here, and that is:

```suggestion
all: docker-build bundle-build
```

Since `bundle` is now a dependency of `bundle-build` a flow has been exemplified down below.